datahub-web-react/src/app/entity/dataset/profile/schema/Schema.tsx

```diff
@@ -16,6 +29,10 @@ const ViewRawButtonContainer = styled.div`
 
 export type Props = {
     schema?: Schema | null;
+    editableSchemaMetadata?: EditableSchemaMetadata | null;
+    updateEditableSchema: (
```

Comment:
Does the schema need to know about the result of this method? If not, can we just use a function that returns void?

Reply:
We do- because we await the promise of the update in order to show/hide a loading bar